docs/source/models/concept_models/vivarium_moud/concept_model.rst

```diff
@@ -131,23 +138,124 @@ Key features of the model include:
 - Treatment state has no excess mortality or disability weight relative to susceptible
 - State membership is determined by prevalence data and treatment ratios designed to be internally consistent and to match GBD 2021 estimates
 
+3.2 Quarters Model
+~~~~~~~~~~~~~~~~~~
+
+The simulation also includes a "Quarters Model" representing the living situation of each individual. This submodel tracks transitions between distinct states related to housing, homelessness, and incarceration.
+
+.. graphviz::
+
+   digraph quarters_model {
+       rankdir = TD; // Suggesting Top-Down layout for potential clarity
+       node [shape=box]; // Optional: make nodes boxes like states
+
+       private_residence [label="Private Residence"];
+       unhoused [label="Unhoused"];
+       incarcerated [label="Incarcerated"];
+
+       // Transitions out of Private Residence
+       private_residence -> unhoused [label="pr_uh"];
+       private_residence -> incarcerated [label="pr_inc"];
+
+       // Transitions out of Unhoused
+       unhoused -> private_residence [label="uh_pr"];
+       unhoused -> incarcerated [label="uh_inc"];
+
+       // Transitions out of Incarcerated
+       incarcerated -> private_residence [label="inc_pr"];
+       incarcerated -> unhoused [label="inc_uh"];
+   }
+
+This submodel uses a state machine with three states:
+
+.. list-table:: State Definitions
+   :widths: 15 30
+   :header-rows: 1
+
+   * - State
+     - Definition
+   * - Private Residence
+     - Individual resides in stable, private housing (e.g., house, apartment).
+   * - Unhoused
+     - Individual lacks stable housing (e.g., living on streets, in shelters, temporary arrangements).
+   * - Incarcerated
+     - Individual resides in a correctional facility (e.g., jail, prison).
```

Comment:
Above you mention recovery housing. Will this not be included as a state in the model?

Reply:
I did not think about this before.  I guess it might be included explicitly with an additional state in the quarters model, or it might be included similarly to a RiskEffect that makes it more likely to stay in the "Private Residence" state as well as to stay in the "On Treatment" state.  To be investigated further if/when we develop that intervention model!